plugin-locationcomponent/src/main/java/com/mapbox/maps/plugin/locationcomponent/LocationPuckManager.kt

```diff
@@ -148,11 +148,17 @@ internal class LocationPuckManager(
     locationLayerRenderer.hide()
   }
 
+  /**
+   * function to set scaling for [LocationPuck].
+   * In order to keep 3D puck size constant across all zoom levels, we interpolate the model based on
+   * current zoom level. MIN_ZOOM, MAX_ZOOM are used as two anchor points to calculate
+   * the scale expression.
+   */
   @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
   internal fun styleScaling(settings: LocationComponentSettings) {
     val puck = settings.locationPuck
-    val minZoom = delegateProvider.mapCameraManagerDelegate.getBounds().minZoom
-    val maxZoom = delegateProvider.mapCameraManagerDelegate.getBounds().maxZoom
+    val minZoom = MIN_ZOOM
+    val maxZoom = MAX_ZOOM
```

Comment:
Shall we add some tests to cover this change?

Reply:
default 3d scale test covers these.